includes/ECommerce.php

```diff
@@ -543,4 +549,80 @@ public function detect_plugin_activation( $plugin ) {
+			if ( ! get_user_meta( get_current_user_id(), 'manageedit-pagecolumnshidden' ) ) {
+				update_user_meta( get_current_user_id(), 'manageedit-pagecolumnshidden', array( 'author', 'comments', 'date', 'wpseo-score', 'wpseo-score-readability', 'wpseo-title', 'wpseo-metadesc', 'wpseo-focuskw', 'wpseo-links' ) );
+			}
+			if ( ! get_user_meta( get_current_user_id(), 'manageedit-postcolumnshidden' ) ) {
+				update_user_meta( get_current_user_id(), 'manageedit-postcolumnshidden', array( 'author', 'categories', 'tags', 'comments', 'date', 'wpseo-score', 'wpseo-score-readability', 'wpseo-title', 'wpseo-metadesc', 'wpseo-focuskw', 'wpseo-links' ) );
+			}
+		}
+	}
+
+	/**
+	 * Add custom column header for post/page/product screen
+	 *
+	 * @param array $columns Array of column names for posts/pages
+	 */
+	public function custom_status_column( $columns ) {
+		if ( 'product' != get_post_type() && 1 == get_option( 'onboarding_experience_level' ) ) {
+			// Add 'Status' column after 'Title'
+			$columns['status'] = __( 'Status', 'wp-module-ecommerce' );
+		}
+		return $columns;
+	}
+
+	/**
+	 * Shows status and availability under status
+	 *
+	 * @param string $column_name column names to which content needs to be updated
+	 *
+	 * @param int    $post_id Id of post/page
+	 */
+	public function custom_status_column_content( $column_name, $post_id ) {
+		if ( __( 'Status', 'wp-module-ecommerce' ) === $column_name ) {
+			// Get the post status
+			$post_status = get_post_status( $post_id );
+			// Get the post date
+			$post_date = get_post_field( 'post_date', $post_id );
+			// Get the post visibility
+			$post_visibility = get_post_field( 'post_password', $post_id );
+
+			$common_style = 'height: 24px; border-radius: 13px 13px 13px 13px;  gap: 16px; padding: 5px 10px; font-weight: 590;font-size: 12px;';
+			if ( 'publish' === $post_status ) {
+				$background_color = empty( $post_visibility ) ? '#C6E8CA' : '#FDE5CC';
+				$label_text       = empty( $post_visibility ) ? __( 'Published - Public', 'wp-module-ecommerce' ) : __( 'Published - Password Protected', 'wp-module-ecommerce' );
+			} elseif ( 'private' === $post_status ) {
+				$background_color = '#CCDCF4';
+				$label_text       = 'Published - Private';
+			} else {
+				$background_color = '#E8ECF0';
+				$label_text       = $post_status;
+			}
+			// Check if coming soon option is enabled
+			$coming_soon = get_option( 'nfd_coming_soon' );
+			if ( $coming_soon ) {
+				$background_color = '#E8ECF0';
+			}
+			echo '<span style="background-color: ' . $background_color . '; ' . $common_style . '">' . $label_text . '</span><br>' . __( 'Last Modified', 'wp-module-ecommerce' ) . ' : ' . mysql2date( 'Y/m/d \a\t g:i a', $post_date );
+		}
+	}
+
+	/**
+	 * Add sorting for the status column
+	 *
+	 * @param array $columns Array of sortable column names for posts/pages
+	 */
+	public function sortable_columns( $columns ) {
+		$columns['status'] = 'status';
```

Comment:
Add translation tag

Reply:
This is a column name. Cannot add the translation for this.